playbooks/common/openshift-node/configure_nodes.yml

```diff
@@ -13,5 +13,6 @@
   roles:
   - role: os_firewall
   - role: openshift_node
+  - role: openshift_node_dnsmasq
```

Comment:
If it was a dependency previously, then the correct action is to run the role _before_ openshift_node.